src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ItemsControl.cs

```diff
@@ -66,6 +68,55 @@ public async Task When_ContainerSet_Then_ContentShouldBeSet()
 			Assert.AreEqual("item 0", tb.Text);
 		}
 
+		public class ItemForDisplayMemberPath
+		{
+			public string DisplayName { get; set; }
+			public override string ToString() => "This is .ToString() result - should not be used";
+		}
+
+		[TestMethod]
+		[RunsOnUIThread]
+		public async Task When_SpecifyingDisplayMemberPath_Then_ContentShouldBeSet()
+		{
+			var resources = new TestsResources();
+
+			var SUT = new ItemsControl
+			{
+				ItemTemplate = TextBlockItemTemplate,
+				DisplayMemberPath = "DisplayName"
+			};
+
+			WindowHelper.WindowContent = SUT;
+
+			await WindowHelper.WaitForIdle();
+
+			var source = new[]
+			{
+				new ItemForDisplayMemberPath {DisplayName = "item 0"},
+				new ItemForDisplayMemberPath {DisplayName = "item 1"},
+				new ItemForDisplayMemberPath {DisplayName = "item 2"}
+			};
+
+			SUT.ItemsSource = source;
+
+
+			async Task Assert(int index, string s)
```

Comment:
Codacy found an issue: [Change return type to 'void'; not a single caller uses the returned value.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6175348)

Reply:
Oooh... that's a real bug.  I'll fix it.